LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License Copyright (c) 2022 Dan Fishgold
```

Comment:
Does it have to be a person or can it be Code Reading Club CIC ?

I mean I know it does not exist yet, but... I don't think that matters or if it does, then leave the CIC off - we are informally an established organisational body

Reply:
I mean unless you want the (c) which is also fine.

Reply:
the plan was to change it once we set up the CIC but I'll do that now :)